app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarMenuController.kt

```diff
@@ -295,6 +297,16 @@ class DefaultBrowserToolbarMenuController(
         }
     }
 
+    private fun getProperUrl(currentSession: Session?): String? {
```

Comment:
Ah we already addressed this in a few places. Best thing to do is get the original / article URL from the store e.g.:
https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/collections/CollectionCreationStore.kt#L98

This way you don't have to parse the URL which may break in the future.

Here you can get to it via `currentSession?.toTabSessionState()?.readerState` for now until we refactor this class to use the `BrowserStore` instead of `SessionManager`.